### PR TITLE
[12.0][IMP] hr_attendance_modification_tracking: ADD 'time changed' -contains all commits-

### PR DESCRIPTION
**Pull request by ns-landoo**
This PR contains all commits included in the all one  https://github.com/OCA/hr/pull/837

### CODE REVIEW

**Review by jarroyomorales — APPROVED** [review]
LGTM! It could be squashed in the other PR but its ok anyways. Thank you!

@Saran440  Can you check?

**Review by Saran440 — APPROVED** [review]
Functional Test. LGTM :+1:

File: hr_attendance_modification_tracking/__manifest__.py [hunk 1]
```diff
@@ -1,13 +1,14 @@
...
+# Copyright 2020 Landoo
 # License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
 
 {
-    'name': 'Hr Attendance Modification Tracking',
+    'name': 'HR Attendance Modification Tracking',
     'summary': """
         Attendance changes will now be registered in the chatter.""",
-    'version': '12.0.1.0.0',
+    'version': '12.0.1.1.0',
```
Comment (alexey-pelykh) [inline hr_attendance_modification_tracking/__manifest__.py:9]: Optional, as it's being taken care of during merge

**Review by alexey-pelykh — APPROVED** [review]

File: hr_attendance_modification_tracking/__manifest__.py (hunk 1 above)
Comment (js-landoo) [reply hr_attendance_modification_tracking/__manifest__.py:9]: I agree. Can we change it without starting again with all revision process?

**Comment by js-landoo** [comment]
@etobella @jarroyomorales and others. Could you merge this PR?
This way we could start with spanish translation. There is an issue with po file. It says is version (11).
Thanks.

File: hr_attendance_modification_tracking/__manifest__.py (hunk 1 above)
Comment (Saran440) [reply hr_attendance_modification_tracking/__manifest__.py:9]: @alexey-pelykh this case, Can I merge this pr with nobump?

**Review by marcelsavegnago — APPROVED** [review]

**Review by etobella — APPROVED** [review]
LGTM :+1: @OCA/human-resources-maintainers I think this can be merged.

**Pull request merged**